examples/text_embeddings_v2/export_v2.py

```diff
@@ -176,10 +176,10 @@ def __call__(self, sentences):
 
 
 def export_module_from_file(embedding_file,
+                            num_oov_buckets,
```

Comment:
Please leave the defaults. That means leaving it in original order, but changing the at call, i.e.

export_module_from_file(FLAGS.embedding_file, FLAGS.export_path,
                                         FLAGS.num_oov_buckets, FLAGS.num_lines_to_ignore,
                                         FLAGS.num_lines_to_use)

To prevent this happening in the future, it is a good practice to call function with keywords:
export_module_from_file(embedding_file=FLAGS.embedding_file, 
                                         export_path=FLAGS.export_path,
                                         num_oov_buckets=FLAGS.num_oov_buckets, 
                                         num_lines_to_ignore=FLAGS.num_lines_to_ignore,
                                         num_lines_to_use=FLAGS.num_lines_to_use)

Reply:
The defaults are in argparse. 
![image](https://user-images.githubusercontent.com/17886829/60660558-d7302980-9e79-11e9-89a4-fa9612727793.png)

Should I change to using keywords now? Or leave it like this since now the orders are fixed.

Reply:
Ok.